arvo/clay/data-types.md

```diff
@@ -216,91 +281,68 @@ that was at the head on date `p`, `%tas` refers to the commit labeled
-
-`ang` is unused and should be removed.
+A `dome` is the state of a `desk` and associated data.
 
 `ank` is the current state of the desk. Thus, it is the state of the
-filesystem at revison `let`. The head of a desk is always a numbered
+filesystem at revison `let`. The head of a `desk` is always a numbered
 commit.
 
 `let` is the number of the most recently numbered commit. This is also
 the total number of numbered commits.
 
-`hit` is a map of numerical ids to hashes of commits. These hashes are
-mapped into their associated commits in `hut:rang:clay`. In general, the keys
-of this map are exactly the numbers from 1 to `let`, with no gaps. Of
-course, when there are no numbered commits, `let` is 0, so `hit` is
-null. Additionally, each of the commits is an ancestor of every commit
-numbered greater than this one. Thus, each is a descendant of every
-commit numbered less than this one. Since it is true that the date in
-each commit (`t:yaki`) is no earlier than that of each of its parents,
-the numbered commits are totally ordered in the same way by both
-pedigree and date. Of course, not every commit is numbered. If that
-sounds too complicated to you, don't worry about it. It basically
-behaves exactly as you would expect.
+`hit` is a map of numerical ids to commit hashes. These hashes are mapped into
+their associated commits in `hut.rang` in the `raft` of Clay. In general, the
+keys of this map are exactly the numbers from 1 to `let`, with no gaps. Of
+course, when there are no numbered commits, `let` is 0, so `hit` is null.
+Additionally, each of the commits is an ancestor of every commit numbered
+greater than this one. Thus, each is a descendant of every commit numbered less
+than this one. Since it is true that the date in each commit (`t:yaki`) is no
+earlier than that of each of its parents, the numbered commits are totally
+ordered in the same way by both pedigree and date. Of course, not every commit
+is numbered. If that sounds too complicated to you, don't worry about it. It
+basically behaves exactly as you would expect.
 
 `lab` is a map of textual labels to numbered commits. Note that labels
 can only be applied to numbered commits. Labels must be unique across a
 desk.
 
-#### `++ankh`, filesystem node
+`mim` is a cache of the content in the directories that are mounted to Unix.
+Often, we convert to/from mime without anything really having changed; this lets
+us short-circuit that in some cases. Whenever you `%give` an `%ergo` `gift`, you
+must update this.
 
-```hoon
-    ++  ankh                                                ::  fs node (new)
-              $:  p=cash                                    ::  recursive hash
-                  q=(unit ,[p=cash q=*])                    ::  file
-                  r=(map ,@ta ankh)                         ::  folders
-              ==                                            ::
-```
+`fod` is the Ford cache, which keeps files, `mark`s, `cast`s, and `tube`s
```

Comment:
I'd describe this in terms of build results rather than individual data structures.  The ford cache maintains a cache of the results of builds performed at this desk's current revision, including the full transitive closure of dependencies for each completed build.

Reply:
https://github.com/urbit/docs/pull/1055/commits/4957fff63755abf87fd80b1e3ec77e79f54c0323